frontend/src/scenes/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Formik, Form, useField } from 'formik';
+import * as Yup from 'yup';
+import './SignUp.sass';
+import { login } from '../../services/authService';
+import { setLocalStorageItem } from '../../helpers/localStorageHelper';
+
+const InputField = ({ label, ...props }: { label: string, name: string, type: string, placeholder?: string }) => {
```

Comment:
You define the same InputField component in 3 different files. 
Please, generalize it, move this definition to separate file and use everywhere you need InputField